splunk_add_on_ucc_framework/commands/build.py

```diff
@@ -655,6 +655,8 @@ def generate(
                 "additional_packaging.py is present but does not have `additional_packaging`. "
                 "Skipping additional packaging."
             )
+        # clean-up sys.path manipulation
+        sys.path.pop(0)
```

Comment:
Now I think that maybe instead of modifying `sys.path`, we should use `importlib`?

Reply:
Yes, we can update the `sys.path.insert` to use `importlib` and then remove this `sys.path.pop` method in future PRs?

Reply:
Sound good